rpcs3/Emu/Cell/PPUThread.cpp

```diff
@@ -584,9 +666,21 @@ void ppu_thread::exec_task()
 
 	while (true)
 	{
+		// const auto exec_op = [this](u64 op)
+		// {
+		// 	return reinterpret_cast<func_t>(op & 0xffffffff)(*this, { static_cast<u32>(op >> 32) });
+		// };
 		const auto exec_op = [this](u64 op)
 		{
-			return reinterpret_cast<func_t>(op & 0xffffffff)(*this, {static_cast<u32>(op >> 32)});
+			const auto c = reinterpret_cast<func_t>(op & 0xffffffff)(*this, { static_cast<u32>(op >> 32) });
+			static u32 count = 0;
+			for (auto i = 0; i <32;++i){
+				if ((gpr[i] == 0x58244b46 || gpr[i] == 0x464b2458) && (++count % 2)) [[unlikely]] {
```

Comment:
drop this change(s), I guess its a rebase mistake.

Reply:
fuk, it's a push mistake